.github/workflows/cmake.yml

```diff
@@ -50,7 +59,7 @@ jobs:
            -DCMAKE_CXX_COMPILER=${{ matrix.compiler }}
            -DCMAKE_VERBOSE_MAKEFILE=on
            -DOmega_h_USE_MPI=off
-           -DOmega_h_USE_CUDA=off
+           -DOmega_h_USE_CUDA=${{ matrix.kokkos == 'off' && matrix.cuda || 'off' }}
```

Comment:
nice!

Reply:
No, it is a 'ternary operator'. If Kokkos is disabled then `Omega_h_USE_CUDA` is set to the value of `matrix.cuda` possibly `on` or `off`. Otherwise, if Kokkos is enabled, then `Omega_h_USE_CUDA` is set to `off`. Note, that in this case, Kokkos may or may not be built with CUDA. See: https://docs.github.com/en/actions/learn-github-actions/expressions#example

Reply:
Great.  Thanks for the link and clarifying.  That is tough syntax coming from C/C++/Bash :)